DSS/astm-uam-psu-dss-0.0.0.yaml

```diff
@@ -2169,9 +2226,59 @@ components:
             All volumes of the resource must be encompassed in these extents. However, these extents do not need to match the precise volumes of the resource; a single bounding extent may be provided instead, for instance.
           items:
             $ref: '#/components/schemas/Volume4D'
```

Comment:
"extents" is defined as an array of Volume4Ds. Does this also apply to point-based resources?